Buildings/Resources/src/fmi-library/CMakeLists.txt

```
@@ -7,7 +7,7 @@ cmake_policy(SET CMP0048 NEW)
 cmake_minimum_required(VERSION 3.8)
 
 project( fmi-library
-  VERSION 2.2
+  VERSION 3.0.2
```

```suggestion
  VERSION 3.0.3
```
To fix the issue with the test case visible with newer MSVC compilers.